### PR TITLE
Add a QtSpim Flatpak

### PR DESCRIPTION
**Pull request by refi64**
Largely just created this since both me and someone else at my university on Fedora need to use QtSpim for computer org class, but there's nothing available other than a deb file.

Asked upstream but there has been no reply yet.

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build net.sourceforge.QtSpim

File: net.sourceforge.QtSpim.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+app-id: net.sourceforge.QtSpim
+runtime: org.kde.Platform
+runtime-version: '5.13'
+sdk: org.kde.Sdk
+command: qtspim
+finish-args:
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=x11'
+  - '--socket=wayland'
```
Comment (nedrichards) [inline net.sourceforge.QtSpim.yaml]: needs --device=dri to fix the usual Qt GPU accel:
```
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: Version 4 or later of flush extension not found
libGL error: failed to load driver: i915
libGL error: failed to open drm device: No such file or directory
libGL error: failed to load driver: i965
```

**Comment by nedrichards** [comment]
Has some non-ideal HiDPI behaviour in some places but that's not a blocker. Should send the appdata file upstream ideally, but otherwise only waiting on upstream yea/nay.

**Comment by refi64** [comment]
@nedrichards done

**Comment by LeandroStanger** [comment]
bot, build net.sourceforge.QtSpim

**Comment by refi64** [comment]
No response yet, what would the chances be of adding this now and then
potentially transferring ownership later if needed?

**Pull request closed without merging**